xbmc/dbwrappers/dataset.cpp

```diff
@@ -270,76 +264,114 @@ bool Dataset::goto_rec(int pos) {
   return false;
 }
 
-
 void Dataset::insert() {
-   edit_object->resize(field_count());
-   for (int i=0; i<field_count(); i++) {
-     (*fields_object)[i].val = "";
-     (*edit_object)[i].val = "";
-     (*edit_object)[i].props = (*fields_object)[i].props;
-   }
+  edit_object->resize(field_count());
+  for (int i=0; i<field_count(); i++) {
```

Comment:
could you please put brackets to new line here and everywhere else (adjust you formatter), thanks